src/pages/api/events.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+export default async (req, res) => {
```

Comment:
using lambdas solves for a host of CORS issues

Reply:
I ❤️ vercel's approach to this. So simple.